src/app/profiles/i18n/index.ts

```diff
@@ -0,0 +1,30 @@
+import i18n from 'i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import { initReactI18next } from 'react-i18next';
+import resources from '../../../locales/resources';
+import format from './format';
+
+const options = {
+  resources,
```

Comment:
We might find a way to share most options.

Reply:
i will do that in #825